### PR TITLE
Moved Service attributes to registry

### PR DESCRIPTION
**Pull request by gregkalapos**
## Changes

This PR is to moves `service` and `service_experimental` attributes to the attributes registry and make service resources reference the registry.

## Merge requirement checklist

* [x] [CONTRIBUTING.md](https://github.com/open-telemetry/semantic-conventions/blob/main/CONTRIBUTING.md) guidelines followed.
* [ ] Change log entry added, according to the guidelines in [When to add a changelog entry](https://github.com/open-telemetry/semantic-conventions/blob/main/CONTRIBUTING.md#when-to-add-a-changelog-entry).
  * If your PR does not need a change log, start the PR title with `[chore]`
* [ ] [schema-next.yaml](https://github.com/open-telemetry/semantic-conventions/blob/main/schema-next.yaml) updated with changes to existing conventions.

### CODE REVIEW

**Review by joaopgrassi — APPROVED** [review]

**Review by pyohannes — APPROVED** [review]

**Comment by joaopgrassi** [comment]
@gregkalapos can you please fix the yamllint issues? then we should be good to merge :)

**Review by jsuereth — APPROVED** [review]

File: docs/resource/README.md
Comment (trisch-me) [inline docs/resource/README.md:85]: I'm a bit concerned that we are not rendering `stable` attribute anymore unless it's a registry

Comment (lmolkova) [reply docs/resource/README.md:85]: It's a big, fixing it here https://github.com/open-telemetry/build-tools/pull/272

Comment (joaopgrassi) [reply docs/resource/README.md:85]: hum I guess we should wait then to merge this? Given the missing stability badge. Good catch @trisch-me !

Comment (gregkalapos) [reply docs/resource/README.md:85]: I'm watching https://github.com/open-telemetry/build-tools/pull/272 - once it's merged, I'll rebase and regenerate the `.md` files.

Comment (trisch-me) [reply docs/resource/README.md:85]: @gregkalapos PR is merged

Comment (gregkalapos) [reply docs/resource/README.md:85]: How can I use a version with the fix-PR of the `md` file generation?

From what I see [`make table-generation`](https://github.com/open-telemetry/semantic-conventions/blob/main/Makefile#L97) uses the [`otel/semconvgen` image](https://hub.docker.com/r/otel/semconvgen/tags). v0.23.0 definitely doesn't have the PR, tried latest but that does not generate the `![Stable]` part either.

Does anyone need to push an updated version of the docker image? Or can I do something locally myself?

Comment (joaopgrassi) [reply docs/resource/README.md:85]: We need to wait for a release of the build tools to be published, to ultimately merge. I guess you can still do it by building it locally and changing the make target to use your local image but we won't merge it the official one is out. I think we will do that soon (hopefully)

**Comment by trisch-me** [comment]
@gregkalapos could you please update this PR?

**Comment by gregkalapos** [comment]
This is still pending on https://github.com/open-telemetry/semantic-conventions/pull/820. I think https://github.com/open-telemetry/semantic-conventions/pull/820 should be merged first and then I could update this.

Alternatively I could use the changed from https://github.com/open-telemetry/semantic-conventions/pull/820, update this PR and maybe we could merge - I personally don't think that'd be ok, but happy to do so if others think it makes sense.

Let me know how to proceed - otherwise I'll just wait for https://github.com/open-telemetry/semantic-conventions/pull/820.

**Comment by gregkalapos** [comment]
https://github.com/open-telemetry/semantic-conventions/pull/820 merged, I updated the PR.

**Comment by gregkalapos** [comment]
Currently fighting with [areas-dropdown-check](https://github.com/gregkalapos/semantic-conventions/actions/runs/8473453798/job/23217797550).

```
`Dropdowns in issue templates is out of date, please run "make generate-gh-issue-templates" and commit the changes in this PR.
Error: Process completed with exit code 1.
```

`make generate-gh-issue-templates` fails for me locally - trying to figure this out... 😢

**Review by joaopgrassi — APPROVED** [review]

**Comment by trisch-me** [comment]
@gregkalapos I had the same issue, have created a hint for it
https://github.com/open-telemetry/semantic-conventions/pull/876/files
@joaopgrassi FYI

**Pull request merged**
